dgbpy/deeplearning_apply_serverlib.py

```diff
@@ -145,6 +145,7 @@ def preprocess(self,samples):
         elif dgbhdf5.applyRangeScaling( self.info_ ):
             self.scaler_ = dgbscikit.getNewRangeScaler( samples )
         elif dgbhdf5.applyGlobalStd( self.info_ ):
+            self.scaler_ = dgbscikit.getScaler( samples, True )
```

Comment:
Should not be required.
@hpratama can you confirm that ModelApplier.setOutput is not executed for this log clustering workflow, as it suggests? This function is doing much more than just receiving the output names, and should also be executed when predicting from an unsupervised model